.github/actions/link-package/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Build Lumino from source and link to JupyterLab
+description: |
+  Build Lumino from source then link to JupyterLab.
+  Useful to test changes in Lumino against a given build of JupyterLab.
+
+inputs:
+  external_pkg_repo:
+    description: |
+      External package repository to link, for example jupyter/lumino
+      only to be used if you want to link and test a package against JupyterLab.
+      Otherwise leave blank.
+    required: false
+    default: ""
+  external_pkg_ref:
+    description: |
+      External package repository's ref (branch, tag or SHA) to checkout,
+      only to be used if you want to link and test a package against JupyterLab.
+      Otherwise leave blank.
+    required: false
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout package repo 🛎
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.external_pkg_repo }}
+        ref: ${{ inputs.external_pkg_ref }}
+        path: external-pkg
+
+    # following https://github.com/jupyterlab/lumino/blob/main/CONTRIBUTING.md#setting-up-a-development-environment
+    - name: Build package from source 🏗
+      run: |
+        yarn
+        yarn build:src
+        conda run --prefix $CONDA_PREFIX jlpm link
```

Comment:
Do we need to add some note somewhere in this file that this action needs to be run in a certain context, like within a workflow that sets up `yarn` and `conda`? I mean, like, some way to indicate that this is very much not a standalone action.

Reply:
Added now